admin/test/config/database.yml

```diff
@@ -0,0 +1,18 @@
+localhost:
+  username: 'sa'
+  password: 'Mtc-D3v.5ql_S3rv3r'
+  host: 'localhost'
+  port: '1433'
+  database: 'mtc'
+dev:
```

Comment:
i would just have one set that default to the local dev instance values, but can be overridden by env vars.
see `./admin/config.js` for inspiration

Reply:
Done